src/localkiwixserver.cpp

```diff
@@ -32,8 +32,12 @@ LocalKiwixServer::LocalKiwixServer(QWidget *parent) :
             break;
         }
     }
+    ui->KiwixServerButton->setStyleSheet("QPushButton {background-color: RoyalBlue;"
+                                                      "color: white;"
+                                                      "padding: 5px;"
+                                                      "border-radius: 3px;}"
+                                         "QPushButton:hover {background-color: DodgerBlue;}");
```

Comment:
It would be nice to have this css in a separated css file.
It is possible to add specific property to widget and use QCSS selector for this property : https://forum.qt.io/topic/85946/qt-style-sheets-cascading-classes-selectors/2

Reply:
@HanYoung-uwu any feedback here?